README.md

```diff
@@ -122,11 +186,13 @@ the few examples we will cover in the rest of this section:
 }
 ```
 
-With this configuration, I have all my project files in a directory called `qad`
-(which stands for "Quick And Dirty", which is a directory name I used when I
-just want to experiment with some files). In that directory, I have generated my
-based `tsconfig.json` and have also created a `src` directory with all my `.ts`
-files and a `dist` directory that will hold all my generated `.js` files. My
-file structure then looks like this:
+Take note that we've specified the `rootDir` as `src` and the `ourDir` as `dist`. 
```

Comment:
Typo: `ourDir` (I know you said I could ignore this part for now but I happened to notice it 🙂 )